app/views/submissions/_annotation_global.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="base-global-annotation-form global-annotation-form">
```

Comment:
May need to add a `display: none` to solve "Global annotation" header text flashing
![image](https://user-images.githubusercontent.com/50491000/204153693-1614ee8e-f6f3-46b1-bf89-2cf9655a82ca.png)

Reply:
Done